docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,315 @@
+What will you get?
+------------------
+.. image:: ../_static/images/integrations/lightning_basic.gif
+   :target: ../_static/images/integrations/lightning_basic.gif
    :alt: PyTorchLightning neptune.ai integration
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
+
+* see experiment as it is running,
+* log training and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
+
+.. note::
+
+    This integration is tested with ``pytorch-lightning==0.9.0``. See 'Troubleshoot' section for information about other versions.
+
+Quickstart
+----------
+This quickstart will show you how to log PyTorch Lightning experiments to Neptune using ``NeptuneLogger`` (part of the pytorch-lightning library).
+
+Check this public project with example experiments: |project|.
+
+.. tip::
+
+    Follow whatever is most convenient to you:
+
+    1. This page for detailed explanations,
+    2. Open quickstart code in Colab Notebook (badge-link below) and run it as a "`neptuner`" user - zero setup, it just works,
+    3. View quickstart code as Python script on |script|.
+
+|Run on Colab|
+
+Before you start
+^^^^^^^^^^^^^^^^
+**Prerequisites**
+
+* Python 3,
+* Neptune account |register|,
+* ``pytorch-lightning==0.9.0`` and ``torchvision`` installed (lightning |lightning-install|),
+* Minimal familiarity with the PyTorch Lightning.
+
+Installation
+^^^^^^^^^^^^
+**Install neptune-client**
+
+From PyPI:
+
+.. code-block:: bash
+
+    pip install neptune-client -U
+
+From Conda:
 
 .. code-block:: bash
 
-    pip install neptune-client
+    conda install neptune-client -c conda-forge
+
+Now, you have all dependencies installed. Let's move to the actual integration.
+
+Step 1: Import Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^
+Import necessary libraries.
 
+.. code-block:: python3
+
+    import os
+
+    import torch
+    from torch.nn import functional as F
+    from torch.utils.data import DataLoader
+    from torchvision.datasets import MNIST
+    from torchvision import transforms
+
+    import pytorch_lightning as pl
 
-Create the **LightningModule**
-------------------------------
+Notice ``pytorch_lightning`` at the bottom.
 
-Nothing changes here, just your typical ``LightningModule`` structure.
+Step 2: Define Hyper-Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Define hyper-parameters for model training.
 
 .. code-block:: python3
 
-    class CoolSystem(pl.LightningModule):
+    PARAMS = {'max_epochs': 3,
+              'learning_rate': 0.005,
+              'batch_size': 32}
+
+Hyper-parameters are passed in the regular Python dictionary. You will see them logged in Neptune parameters tab.
```

Comment:
are passed to what -> in this section the reader doesn't yet know how the logger looks or that it exists really. 
This should be more explicit with less assumptions about what reader knows I feel.

Reply:
Agree, that User does not know much about logger at this point. I wanted co communicate that you just have prepared params in python dict -> code above is just this.
I need to change wording or edit these two sentences and it will be clear :)